steps/50-lifecycle-lab2-dist/.github/workflows/release-workflow.yml

```
@@ -11,6 +11,7 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
```

```suggestion
    if: github.ref == 'refs/heads/main'
```
Master is deprecated